src/middlewared/middlewared/pytest/target.conf

```diff
@@ -1,4 +1,3 @@
-[Target]
```

Comment:
I think that line was removed by mistake as well?

Reply:
Oh Yeah, I forgot to add it back thanks @william-gr